qfieldsync/core/cloud_api.py

```diff
@@ -734,6 +735,43 @@ def _clear_cloud_cookies(self, url: QUrl) -> None:
             self._nam.cookieJar().deleteCookie(cookie)
 
 
+    def get_localized_datasets_project(self) -> Optional[str]:
+        """
+        Ensures that the 'localized_datasets' project exists under the user's organization.
+
+        Returns:
+            str | None: The project ID of the 'localized_datasets' project, or None if creation failed.
+        """
+        try:
+         
+          
+            username = self.user_details.get("username")
```

Comment:
We can't assume the username is the owner of the project we are uploading files to here, we need to pass it on as a parameter.